src/vxingest/netcdf_to_cb/netcdf_builder.py

```diff
@@ -639,7 +639,37 @@ def umask_value_transform(self, params_dict):
             )
             return None
 
-    def handle_wind_dir_U(self, params_dict):
+    def handle_rh(self, params_dict):
+        """Retrieves a RH value, if there is one,
+           and if not derives the RH from the dewpoint and temperature.
+        Args:
+            params_dict (dict): named function parameters
+        Returns:
+            float: the RH
+        """
+        try:
+            # dewpoint = self.umask_value_transform({"recNum": params_dict["recNum"], "dewpoint":"dewpoint"})
+            # temperature = self.umask_value_transform({"recNum": params_dict["recNum"], "temperature":"temperature"})
+            dewpoint = self.umask_value_transform(
+                {"recNum": params_dict["recNum"], "dewpoint": params_dict["dewpoint"]}
+            )
+            temperature = self.umask_value_transform(
+                {
+                    "recNum": params_dict["recNum"],
+                    "temperature": params_dict["temperature"],
+                }
+            )
+            _q = (
+                relative_humidity_from_dewpoint(
+                    temperature * units.kelvin, dewpoint * units.kelvin
+                ).magnitude
+            ) * 100
```

Comment:
Can we rely on `units.kelvin` for both temperature and dewpoint or will we need to update this as we add new models? Does the NetCDF file provide unit information?

Otherwise, it seems like this is also inconsistent with `backfill_obs_with_rh.calc_components()`. That function is using `units.degC` instead of Kelvin.

Reply:
good catch. The comment is wrong. I'll change it. There aren't any RH values in the METAR netcdf files. The recnum is the data record number (index) into the netcdf data array.

Reply:
There is an issuue to resolve all the non metric units across the board. I'd rather wait and address those all together since they will affect all the builders simultaneously and require backfilling scripts as well.

Reply:
Okay. Is there any concern about inconsistent units in the database? I suspect we're in that situation already. However, I mostly want to make sure that doing the backfilling in Celsius and the realtime ingest in Kelvin won't cause issues.

Reply:
After some offline conversation, it turns out that doing the realtime ingest in Kelvin is correct - the values in these particular (maybe all?) NetCDF files are in degrees Kelvin. However, the backfilling script should have been using degrees Fahrenheit instead of Celsius. The backfilling script operates on the values in the database and those values are stored in degrees Fahrenheit. 

Ideally, in the near future, the raw data files will have whatever units make sense (most likely Kelvin) and the database will store everything in metric.